.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @zeta-chain/protocol-engineering
```

Comment:
Can we rather add handles?
When adding team, it asks the review from a single member of the team (I didn't find option to change this behavior)
We also don't need all team members as reviewers: Stefa, Borja, you and I for now is good. Charlie specializes mainly on ZetaClient and Tanmay on Cosmos SDK

Reply:
https://github.com/zeta-chain/protocol-contracts-ton/pull/15/commits/f45ed6c6498d80297a19405163f1a3dc3dc7a560